open-policy-agent/resource-exhaustion/disallow-self-provisioner/template.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sdisallowselfprovisioner
+  annotations:
+    description: Disallows self provisioner role to all users.
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sDisallowSelfProvisioner
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package k8sdisallowselfprovisioner
+        violation[{"msg": msg}] {
+          input.review.kind.kind == "ClusterRoleBinding"
+          input.review.object.metadata.name == "self-provisioners"
```

Comment:
This suppose to be "self-provisioner" @michaelkotelnikov

Reply:
@igorkchyts the file you commented on is outdated, and is not part of the commit. When merging, please use `squah and merge` on this PR.